Makefile

```diff
@@ -1,6 +1,8 @@
 TEST_INVENTORY ?= tests/inventory.yaml
 TEST_VARS ?= tests/vars.yaml
 TEST_SECRETS ?= tests/secrets.yaml
+TEST_OUTFILE := tests/test_minimal_out_$(shell date +%FT%T%Z).log
```

Comment:
I just realized this should go into gitignore, and i was also wondering whether it would be better in a logs dir so that logs (disposable files) can be more clearly separated from vars and inventory (long-living files, at least in dev envs). But neither of that is a blocker and i'm ok just merging as-is. :+1:

Reply:
o/

No problem, we can create a logs directory. I can push an update here to do that and will update the rdo side to pickup the new file location. I will add the logs dir to the gitignore.

At least this seems to be working OK now... together with the wireup on the rdo side @ https://review.rdoproject.org/r/c/rdo-jobs/+/47392 (where I've now moved ci specific vars/secrets/ansible.cfg) we have the running test with the logs at [1].

[1] https://logserver.rdoproject.org/23/46923/9/check/data-plane-adoption-github-rdo-centos-8-crc-centos-9-standalone/9dc782b/logs/subnode-1/home/zuul-worker/test_minimal_out_2023-03-16T12:25:37UTC.log